src/app/pages/checkout-review/checkout-review/checkout-review.component.ts

```diff
@@ -40,6 +42,28 @@ export class CheckoutReviewComponent implements OnInit {
     this.createOrder.emit();
   }
 
+  private setFields() {
+    return [
+      {
+        type: 'ish-checkout-review-tac-field',
+        key: 'termsAndConditions',
+        templateOptions: {
+          required: true,
```

Comment:
required should not be needed, otherwise the validator message is missing for it

Reply:
It is needed here, because the form will otherwise submit, if the checkbox was not clicked.

Reply:
can be removed